### PR TITLE
Add regression example. #19

### PR DESCRIPTION
**Pull request by dcostea**
## Resolves
This solves issue #19 

## Description
The example is using taxi_fare.csv to exemplify the regression

### CODE REVIEW

File: examples/MLOps.NET.Examples/MLOps.NET.Regression/Program.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+            var trainedModel = trainingPipeline.Fit(testTrainTest.TrainSet);
+
+            Console.WriteLine("Evaluating the model");
+            var predictions = trainedModel.Transform(testTrainTest.TestSet);
+            var metrics = mlContext.Regression.Evaluate(predictions);
+
+            //MLOps: Log Metrics
+            Console.WriteLine("Logging metrics");
+            await mlOpsContext.LogMetricsAsync(runId, metrics);
```
Comment (AnoojNair) [inline examples/MLOps.NET.Examples/MLOps.NET.Regression/Program.cs:47]: @dcostea @aslotte , just wondering is this using an older version of the package?

Comment (aslotte) [reply examples/MLOps.NET.Examples/MLOps.NET.Regression/Program.cs:47]: @AnoojNair yes, it's using the latest released alpha package to nuget.org. I was thinking that once we have this example merged and a couple of more features, we can release a new package and update our examples with additional operations (e.g. set training time, git commit hash, etc.)

File: examples/MLOps.NET.Examples/MLOps.NET.Regression/Program.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.ML;
+using Microsoft.ML.Transforms;
+using MLOps.NET.Regression.Entities;
+using MLOps.NET.SQLite;
+using System;
+
```
Comment (aslotte) [inline examples/MLOps.NET.Examples/MLOps.NET.Regression/Program.cs:6]:
```suggestion
using System.Threading.Tasks;

```

File: examples/MLOps.NET.Examples/MLOps.NET.Regression/Program.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+namespace MLOps.NET.Regression
+{
+    class Program
+    {
+        static async System.Threading.Tasks.Task Main(string[] args)
```
Comment (aslotte) [inline examples/MLOps.NET.Examples/MLOps.NET.Regression/Program.cs:11]:
```suggestion
        static async Task Main(string[] args)
```

**Comment by aslotte** [comment]
Thank you for your contribution @dcostea!

**Pull request merged**